package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "rtMedia",
   "dependencies": {
-    "load-grunt-tasks": "~0.2.1"
+    "load-grunt-tasks": "~0.2.1",
+    "npm": "^5.7.1"
```

Comment:
`npm` locally ?

Reply:
I didn't get this. I just ran `grunt` and `npm install`.

Reply:
yes, please check why its got added

Reply:
I think by mistake, I ran npm update command.
I've removed local `npm` as dependency and re-installed packages.